news/_posts/bands_image_analysis.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Quantifying Electrophoresis Gel Bands with QuPath and Galaxy"
+author: "Reyhaneh Tavakoli-Koopaei"
+contributions:
+  authorship: [reytakop, dianichj]
+tags: [GTN, imaging, electrophoresis, QuPath, Galaxy]
+layout: news
+date: "2025-08-18"
```

Comment:
Not sure if this is defined like this. Usually, your file name would be `2025-08-18-bands_image_analysis.md`.
I check the definition of the metadata here: https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html#news